integrationExamples/gpt/hello_world.html

```diff
@@ -24,9 +24,12 @@
             },
             // Replace this object to test a new Adapter!
             bids: [{
-                bidder: 'appnexus',
+                bidder: 'preciso',
```

Comment:
you don't get to change the hello world to your own, you can add a new integration example